src/App/Containers/Modals/index.js

```diff
@@ -0,0 +1,28 @@
+import React       from 'react';
```

Comment:
@easteregg Regarding the style guide, whenever there are JSX tags in a file the extension should be `.jsx`